server/image_remove_libpod.go

```diff
@@ -0,0 +1,49 @@
+// +build libpod
+
+package server
+
+import (
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"golang.org/x/net/context"
+	pb "k8s.io/kubernetes/pkg/kubelet/apis/cri/runtime/v1alpha2"
+)
+
+// RemoveImage removes the image.
+func (s *Server) RemoveImage(ctx context.Context, req *pb.RemoveImageRequest) (resp *pb.RemoveImageResponse, err error) {
+	const operation = "remove_image"
+	defer func() {
+		recordOperation(operation, time.Now())
+		recordError(operation, err)
+	}()
+
+	logrus.Debugf("RemoveImageRequest: %+v", req)
+
+	image, err := s.ImageRuntime().NewFromLocal(req.Image.Image)
```

Comment:
is this the equivalent or !libpod `ResolveNames` (https://github.com/kubernetes-incubator/cri-o/pull/1669/commits/cc1519c7648a245fa1c751d477a7ec7db7c50ef3#diff-1b1ed2e9f387ee4b59929223987f4478R36)?

we need to normalize the image name to take into account additional registries